### ISSUE TITLE
MazSelect multiple options Not working 

### ISSUE THREAD

**Issue by RadwaMosaad** [LouisMazel/maz-ui-3#17]
I am using "maz-ui": "^3.9.0" & "nuxt": "^3.6.5",
I tried to use multiple feature in MazSelect  But it does not Work
Here is my code 
```
<template>
<MazSelect
            v-model="ageGroupId"
            :options="listOfAgeGroups?.value"
            option-value-key="id"
            :option-label-key="$i18n.locale == 'en' ? 'title' : 'titleAr'"
            :option-input-value-key="$i18n.locale == 'en' ? 'title' : 'titleAr'"
            label="Age Group"
            :size="'sm'"
            listPosition="bottom left"
            multiple
            clearable
          />
   </template>
```
```
   <script setup lang="ts">
   import MazSelect from "maz-ui/components/MazSelect";
   import MazInput from "maz-ui/components/MazInput";

   const listOfAgeGroups = ref<any>();
   const ageGroupId = ref<any>([]);
      </script>
```
          
 I am not adding anything else But it works like a single select option 
Any help ?!

**Comment by LouisMazel** [comment]
Hi @RadwaMosaad,

The feature "multiple" is not available for the moment. I will release it soon, follow this issue: https://github.com/LouisMazel/maz-ui/issues/462

**Comment by LouisMazel** [comment]
The support of the `multiple` option is now available in v3.20.0
